app/forms/creator_settings_form.rb

```diff
@@ -21,26 +20,20 @@ class CreatorSettingsForm
   attr_accessor :success
 
   def save
-    if valid?
```

Comment:
There's no need to manually validate because it's implicitly part of activemodel lifecycle